tests/dialects/test_snowflake.py

```diff
@@ -2277,3 +2277,13 @@ def test_window_function_arg(self):
         self.assertEqual(ast.sql("snowflake"), query)
         self.assertEqual(len(list(ast.find_all(exp.Column))), 1)
         self.assertEqual(window.this.sql("snowflake"), "db.schema.FUNC(a)")
+
+    def test_bitor_function(self):
+        self.validate_identity("SELECT BITOR(a, b) FROM table")
+
+    def test_bit_or_function(self):
+        # BIT_OR it just alias of BITOR, so we can transpile to BITOR only
+
+        self.validate_identity("SELECT BITOR(a, b) FROM table")
+
+        self.validate_identity("SELECT BIT_OR(a, b) FROM table", "SELECT BITOR(a, b) FROM table")
```

Comment:
The 1st and 2nd `validate_identity` tests are repeated here.

Let's instead:
- Remove the 2nd call
- Move the 1st and 3rd tests under `test_snowflake`, the separate test function seems a bit overkill